tests/test_confidence.py

```diff
@@ -54,15 +54,49 @@ def test_confidence_leastsq(data, pars, verbose, capsys):
         assert 'Calculating CI for' in captured.out
 
 
+def test_confidence_pnames(data, pars):
+    """Test if pnames works as expected."""
+    minimizer = lmfit.Minimizer(residual, pars, fcn_args=(data))
+    out = minimizer.leastsq()
+
+    assert_paramval(out.params['a'], 0.1, tol=0.1)
+    assert_paramval(out.params['b'], -2.0, tol=0.1)
+
+    ci = lmfit.conf_interval(minimizer, out, p_names=['a'])
+
```

Comment:
if you want to check whether ```p_names``` worked correctly, shouldn't you actually ```assert``` something after the call to ```conf_interval```? I would think perhaps like:
```
assert 'a' in ci
assert 'b' not in ci
```